webapp/src/components/AssetCard/AssetCard.tsx

```diff
@@ -112,6 +114,9 @@ const AssetCard = (props: Props) => {
         showOrderListedTag={showListedTag}
         showMonospace
       />
+      {isFavoritesEnabled && !rental && !isNFT(asset) ? (
```

Comment:
If it's not an NFT, it can't have a rental, so in theory, this wouldn't be needed. What do you think?
```suggestion
      {isFavoritesEnabled && !isNFT(asset) ? (
```